src/Features/Navigation/Screen.php

```diff
@@ -168,25 +168,58 @@ public static function get_plugin_page( $callback ) {
 	 * Register post type for use in WooCommerce Navigation screens.
 	 *
 	 * @param string $post_type Post type to add.
-	 * @param string $parent_id Parent menu item ID.
+	 * @param array  $args Menu item arguments.
 	 */
-	public static function register_post_type( $post_type, $parent_id ) {
+	public static function register_post_type( $post_type, $args ) {
 		self::$post_types[] = $post_type;
 
 		$post_type_object = get_post_type_object( $post_type );
 
-		if ( ! $post_type_object || ! $post_type_object->show_in_menu || ! $parent_id ) {
+		if ( ! $post_type_object || ! $post_type_object->show_in_menu ) {
 			return;
 		}
 
-		Menu::add_item(
-			array(
-				'parent'     => $parent_id,
-				'title'      => esc_attr( $post_type_object->labels->menu_name ),
-				'capability' => $post_type_object->cap->edit_posts,
-				'id'         => $post_type,
-				'url'        => "edit.php?post_type={$post_type}",
-			)
+		$defaults  = array(
+			'multiple' => false,
+			'parent'   => isset( $args['parent'] ) ? $args['parent'] : Menu::DEFAULT_PARENT,
 		);
+		$menu_args = wp_parse_args( $args, $defaults );
+
+		if ( ! $menu_args['multiple'] ) {
```

Comment:
There's probably a better name for this property.

Reply:
This is used to create multiple menu items for the post type.  Adding `All {post_type}s` and `Add new {post_type}` instead of just a single menu item with `{post_type}`.

Maybe `multiple_labels` is a better name, but open to feedback here.